osf/models/institution.py

```diff
@@ -18,8 +18,19 @@
 logger = logging.getLogger(__name__)
 
 
+class InstitutionManager(models.Manager):
+
+    def get_queryset(self):
+        return super().get_queryset().filter(deactivated__isnull=True)
+
+    def get_all_institutions(self):
```

Comment:
can't we override the queryset `.filter` method directly and have the that run something like.
````
def filter(self, *args, **kwargs):
   kwargs.update({'deactivated__isnull': kwargs.get('deactivated__isnull', True)}
   super().filter(*args, **kwargs)
```
seems like better syntax

Reply:
Considering we *never* want to surface deactivated institutions to users outside of the Admin App, this is the most correct approach. It seamlessly replaces the existing API, including handling indirect cases like `user.affiliated_institutions` for free. Your approach would require us to change every single call (implicit and explicit) to the list of institutions, frequently replacing, say, `Institution.objects.all()` with an awkwardly empty `Institution.objects.filter()`.